lib/vectortilefeature.js

```diff
@@ -164,3 +188,43 @@ VectorTileFeature.prototype.toGeoJSON = function(x, y, z) {
         properties: this.properties
     };
 };
+
+// classifies an array of rings into polygons with outer rings and holes
+
+function classifyRings(rings) {
+    var len = rings.length;
+
+    if (len <= 1) return [rings];
+
+    var polygons = [],
+        polygon,
+        ccw;
+
+    for (var i = 0; i < len; i++) {
+        var area = signedArea(rings[i]);
+        if (area === 0) continue;
+
+        if (!ccw) ccw = area < 0;
```

Comment:
Just tagging so that we don't forget: the condition should be `ccw === undefined`, otherwise classification is buggy in certain cases.